libs/ui/src/global_styles.tsx

```diff
@@ -153,6 +150,13 @@ export const GlobalStyles = createGlobalStyle<GlobalStylesProps>`
     .print-only {
       display: block;
     }
+    .screenshot {
+      @page {
+        size: a4 landscape;
+      }
+      width: 100%;
+      height: 100%;
+    }
```

Comment:
TODO - undo styling changes in this file.

Reply:
undone

Reply:
thanks for cleaning up after me :)